### PR TITLE
doc/radosgw: refine top. man. and not. perf. stats

### PR DESCRIPTION
**Pull request by zdover23**
This commit refines the English in the "Topic Management via CLI" and "Notification Performance Statistics" sections of the notifications.rst file.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/notifications.rst [hunk 1]
```diff
@@ -75,41 +75,43 @@ added when the notification is committed to persistent storage.
 Topic Management via CLI
 ------------------------
 
-Configuration of all topics, associated with a tenant, could be fetched using the following command:
+Fetch the configuration of all topics associated with tenants by running the
+following command:
```
Comment (anthonyeleven) [inline doc/radosgw/notifications.rst:79]: Can more than one tenant be specified comma-separated, or with multiple instances of `--tenant=`?

**Review by anthonyeleven — APPROVED** [review]
requested some clarity, non-blocking.

The periods in the PR title seem stilted

File: doc/radosgw/notifications.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/notifications.rst:79]: This is question for @yuvalif. I'll keep an eye on this PR, and when he answers this question, I'll return to this part of notifications.rst to ingest his answer.

Comment (yuvalif) [reply doc/radosgw/notifications.rst:79]: @zdover23 @anthonyeleven tenant has to be a single value. passing multiple `--tenant` parameters is not supported (I think the last one would override)

**Pull request merged**